lib/ssh.js

```diff
@@ -82,7 +84,7 @@ SSH.prototype._queueCommand = function(index) {
             command.handlers.err(data);
         });
 
-        stream.on('data', function(data, extended) {
```

Comment:
Good catch here. Hadn't realized this pointless variable slipped through the cracks.